pytest.ini

```diff
@@ -1,18 +1,37 @@
 [pytest]
 minversion = 7.0
-testpaths = "sunkit_image" "docs"
-norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunkit_image[/\]_dev" ".jupyter" ".history" "tools" "sunkit_image[\/]extern"
+testpaths =
+    sunkit_image
+    docs
+norecursedirs =
+    .tox
+    build
+    docs/_build
```

Comment:
I think we probably need to keep the `[\/]` to make these work on windows. I wonder if we need to do this in the template as well.

Reply:
I would say so.

Reply:
This list can also be reduced.